front/pages/api/w/[wId]/vaults/[vId]/index.ts

```diff
@@ -184,7 +214,17 @@ async function handler(
         });
       }
 
-      await deleteVault(auth, vault);
+      try {
```

Comment:
`nit`: Let's use `const doDelete = useDeleteVault({ owner });` ?

Reply:
It's a backend function, I don't think we can or should use hooks there, right?

Reply:
Totally right, got lost between files sorry 😅